db/seeds.rb

```diff
@@ -1,32 +1,51 @@
 
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
 2.times do
     ticket_desks = TicketDesk.new(
-        :name=>'cinema',
-        :automated=>true
+        :name=> 'cinema',
+        :automated=> true
         )
     ticket_desks.save!
 end
 4.times do
     ticket_desks = TicketDesk.create([{
         :name=>'cinema',
-        :automated=>false
+        :automated => false
         }])
 end
 volumes = [100, 100, 100, 100, 200, 50, 50, 50, 50, 20]
+age = [6, 10, 12, 18, 65]
+genres = ['comedy', 'criminal', 'western']
+descriptions = ['lots of fun', 'scary, deaths', 'guns, western, cool movies']
+i = 0
+while (i<genres.size)
+    Genre.create(
+        title: genres[i],
+        description: descriptions[i]
+    )
+    i=i+1
+
+end
 i = 0
+
 while (i<volumes.size)
-    cinema_hall = CinemaHall.create([{
-        :volume=>volumes[i]
-        }])
+    @cinema=CinemaHall.create(
+        volume: volumes[i]
+        )
+    @cinema.save!
```

Comment:
`create` already saves it, there is no need to call save again :)

Reply:
:D